test/styles.html

```diff
@@ -99,21 +99,31 @@
     });
 
     suite('focused styles', function() {
-      test('underline and label are colored when input is focused', function(done) {
+      test('label is colored when input is focused', function(done) {
         var input = fixture('label-and-value');
-        var line = Polymer.dom(input.root).querySelector('.underline');
         var label = Polymer.dom(input.root).querySelector('label');
 
-        assert.isFalse(line.classList.contains('is-highlighted'), 'line is not highlighted when input is not focused');
-        assert.isFalse(label.classList.contains('label-is-highlighted'), 'label is not highlighted when input is not focused');
+        assert.isFalse(input.focused);
+
+        // Nothing is coloured and the focus underline isn't showing
+        assert.equal(window.getComputedStyle(label).color, 'rgb(115, 115, 115)');
+        assert.equal(window.getComputedStyle(label, ':after').visibility,
+            'hidden');
 
         MockInteractions.focus(input.$.input);
 
-        requestAnimationFrame(function() {
-          assert.isTrue(line.classList.contains('is-highlighted'), 'line is highlighted when input is focused');
-          assert.isTrue(label.classList.contains('label-is-highlighted'), 'label is highlighted when input is focused');
+        // Wait for the animation to finish
+        setTimeout(function() {
```

Comment:
So in this case it's legit a transform and animation that changes the colour, so async without timeout doesn't work...